packages/upset/src/components/Sidebar.tsx

```diff
@@ -97,33 +106,88 @@ export const Sidebar = () => {
+                    actions.sortBy('Degree', 'Ascending');
+                  }
+                }}
+              >
+                <FormControlLabel
+                  key="Degree"
+                  value="Degree"
+                  label={
+                    <Box
+                      css={itemDivCSS}
+                      onClick={() => {
+                        if (sortBy === 'Degree') {
+                          if (sortByOrder === 'Ascending') {
+                            actions.sortBy('Degree', 'Descending');
+                          } else {
+                            actions.sortBy('Degree', 'Ascending');
+                          }
+                        }
+                      }}
+                    >
+                      <Typography>Degree</Typography>
+                      { sortBy === 'Degree' &&
+                        (sortByOrder === 'Ascending' ?
+                          <ArrowUpward sx={arrowIconCSS} /> :
+                          <ArrowUpward sx={{ ...arrowIconCSS, transform: 'rotate(180deg)' }} />
+                        )
                       }
-                    }}
-                  >
-                    <FormControlLabel
-                      key={sort}
-                      value={sort}
-                      label={sort}
-                      control={<Radio size="small" />}
-                    />
-                    <HelpCircle text={helpText.sorting[sort]} />
-                  </Box>
-                )))}
+                    </Box>
+                  }
+                  control={<Radio size="small" />}
+                />
+                <HelpCircle text={helpText.sorting['Degree']} />
+              </Box>
+              <Box
+                css={itemDivCSS}
+                key="Size"
+                aria-label={helpText.sorting['Size']}
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter') {
+                    actions.sortBy('Size', 'Ascending');
+                  }
+                }}
+              >
+                <FormControlLabel
+                  key="Size"
+                  value="Size"
+                  label={
+                    <Box
+                      css={itemDivCSS}
+                      onClick={() => {
+                        if (sortBy === 'Size') {
+                          if (sortByOrder === 'Ascending') {
+                            actions.sortBy('Size', 'Descending');
+                          } else {
+                            actions.sortBy('Size', 'Ascending');
+                          }
+                        }
+                      }}
+                    >
+                      <Typography>Size</Typography>
+                      { sortBy === 'Size' &&
+                        (sortByOrder === 'Ascending' ?
+                          <ArrowUpward sx={arrowIconCSS} /> :
+                          <ArrowUpward sx={{ ...arrowIconCSS, transform: 'rotate(180deg)' }} />
+                        )
```

Comment:
Let's move the ternary to the CSS?
```suggestion
                        <ArrowUpward sx={{ ...arrowIconCSS, transform: (sortByOrder === 'Ascending' ? '' : 'rotate(180deg)') }} />
```

Reply:
We could do this for both if you think it's cleaner/better for react rendering